ce/marketing/solution-uninstall-order.md

```diff
@@ -108,5 +144,8 @@ Marketing solution uninstall order:
 
 If you see any other "anchor" solutions that start with "MicrosoftDynamics_", you can delete these too. They are probably left over from an earlier version that you upgraded. You can remove these in any order after you've uninstalled the other solutions.
 
+Marketing installation also installs some dependencies that might be used in other applications, those can be uninstalled only if they're not being used by non-marketing application:
```

Comment:
I'd rephrase it to make it clearer to customers:
Dependencies of the Marketing applications are installed alongside the Marketing application. These can only be uninstalled if they are not used by other solutions:

Reply:
Good suggestion -- I'll update